src/pci-gateway/src/main/java/ca/bc/gov/open/jag/pcigateway/api/RedirectController.java

```diff
@@ -69,6 +66,17 @@ public RedirectView localRedirect(HttpServletRequest request) throws MissingServ
 
     }
 
+    private RedirectView redirectRequest(HttpServletRequest request) throws MissingServletRequestParameterException {
+
+        RedirectView redirectView = new RedirectView();
+        redirectView.setUrl(processRequest(request, request.getRequestURI().replace(Keys.PCIGW, Keys.SCRIPTS)).toString());
```

Comment:
seems like this can be done in the process request, the signature is the same for both call to `processRequest(request, request.getRequestURI().replace(Keys.PCIGW, Keys.SCRIPTS)`

Reply:
👍